lib/SILOptimizer/Transforms/FunctionSignatureOpts.cpp

```diff
@@ -979,11 +979,11 @@ static void createArgumentRelease(SILBuilder &Builder, ArgumentDescriptor &AD) {
   if (Arg->getType().isAddress()) {
     assert(AD.PInfo->getConvention() == ParameterConvention::Indirect_In
            && F.getConventions().useLoweredAddresses());
-    Builder.createDestroyAddr(RegularLocation(SourceLoc()),
+    Builder.createDestroyAddr(getCompilerGeneratedLocation(),
```

Comment:
This should possibly be a SILBuilderWithScope, but that's for another commit. Your change is NFC.